cardano-testnet/src/Testnet/Byron.hs

```diff
@@ -69,22 +78,59 @@ defaultTestnetOptions = TestnetOptions
   , securityParam = 10
   , nPoorAddresses = 128
   , totalBalance = 8000000000000000
+  , enableP2P = False
   }
 
 replaceNodeLog :: Int -> String -> String
 replaceNodeLog n s = T.unpack (T.replace "logs/node-0.log" replacement (T.pack s))
   where replacement = T.pack ("logs/node-" <> show @Int n <> ".log")
 
 -- | Rewrite a line in the configuration file
-rewriteConfiguration :: Int -> String -> String
-rewriteConfiguration _ "TraceBlockchainTime: False" = "TraceBlockchainTime: True"
-rewriteConfiguration n s | "logs/node-0.log" `L.isInfixOf` s = replaceNodeLog n s
-rewriteConfiguration _ s = s
+rewriteConfiguration :: Bool -> Int -> String -> String
```

Comment:
I'm just seeing this function for the first time. This seems extremely hacky but looks like something to be fixed in another PR.